mitodl/ocw-hugo-themes
# collect and isolate shared static assets

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1055

#### What's this PR do?
This PR isolates shared static assets such as webpack build output and shared images in a new folder called `static_shared`. This will provide better organization here in the theme as well as in the final output of Hugo builds. It will allow easier and more efficient copying of shared static resources into the offline course site bundles as well as prevent images not used by the `course-v2` theme and only by the `www` theme from being included, as it separates them out into the `www` theme's static images folder.

#### How should this be manually tested?
 - Run `yarn start www` to start `ocw-www`
 - Verify that there are no broken images, styles load correctly and there are no errors in the console
 - Run `yarn start course` to start a course site
 - Verify that there are no broken images, styles load correctly and there are no errors in the console
